dbschema/project.esdl

```diff
@@ -57,7 +57,7 @@ module default {
     
     trigger confirmProjectSens after update for each do (
```

Comment:
Personal pet peeve...sensitivity is spelled out 90% of the places...why not make it consistent and spell it out everywhere?  It doesn't add much space to do so.

Reply:
Does that mean you don't like this either? 🙈 
https://github.com/SeedCompany/cord-api-v3/blob/e2df0a2c28559b1250e99c06d44255ed74dd1106/dbschema/sensitivity.esdl#L5-L5

I can spell out.

Reply:
I don't, but there's more of a reason to do it there.  Feel free to ignore.